examples/simple_xo/src/main.rs

```diff
@@ -0,0 +1,246 @@
+use sawtooth_xo::handler::XoTransactionHandler;
+use transact::context::manager::sync::ContextManager;
+use transact::database::{btree::BTreeDatabase, Database};
+use transact::execution::{adapter::static_adapter::StaticExecutionAdapter, executor::Executor};
+use transact::protocol::receipt::TransactionReceipt;
+use transact::protocol::{
+    batch::{BatchBuilder, BatchPair},
+    receipt::StateChange,
+    transaction::{HashMethod, TransactionBuilder},
+};
+use transact::sawtooth::SawtoothToTransactHandlerAdapter;
+use transact::scheduler::{
+    serial::SerialScheduler, BatchExecutionResult, Scheduler, TransactionExecutionResult,
+};
+use transact::signing::{hash::HashSigner, Signer};
+use transact::state::merkle::{self, MerkleRadixTree, MerkleState};
+use transact::state::StateChange as ChangeSet;
+use transact::state::Write;
+
+use hex;
+use sha2::{Digest, Sha512};
+use std::io;
+use std::str;
+use std::sync::{Arc, Mutex};
```

Comment:
We should be able to write this example without Arc/Mutex. But, I am also comfortable with this version going in and then iterating on it as an overall team.

Reply:
I can see why this was done (based on the tests), but it definitely works without.

Reply:
Because there's one instance of executor and the execution happen serially?